libs/components/tabs/src/lib/modules/tabs/tabset.component.spec.ts

```diff
@@ -1574,4 +1593,40 @@ describe('Tabset component', () => {
       validateTabSelected(fixture.nativeElement, 0);
     }));
   });
+
+  describe('Permalinks with non-base URLs', () => {
+    let fixture: ComponentFixture<SkyTabsetPermalinksFixtureComponent>;
+    let location: Location;
+
+    beforeEach(() => {
+      location = TestBed.inject(Location);
+      fixture = TestBed.createComponent(SkyTabsetPermalinksFixtureComponent);
+    });
+
+    it('should not affect the base URL when opened or closed', fakeAsync(() => {
```

Comment:
If you choose to move the spec to the "Permalinks" `describe` block, you could rename this to: "should not affect the current route when destroyed".

Reply:
done